android/app/src/main/java/updated/mysterium/vpn/network/usecase/BalanceUseCase.kt

```diff
@@ -1,13 +1,16 @@
 package updated.mysterium.vpn.network.usecase
 
+import android.content.Context
 import mysterium.GetBalanceRequest
 import network.mysterium.service.core.DeferredNode
 import network.mysterium.service.core.NodeRepository
 
-class BalanceUseCase(private val nodeRepository: NodeRepository) {
+class BalanceUseCase(private val nodeRepository: NodeRepository, private val context: Context) {
 
```

Comment:
Try to not pass context to Usecase. You can make it by creating singletone class for shared preferences, and pass application context when initializing that module

Reply:
Fixed